stdlib/InteractiveUtils/test/runtests.jl

```diff
@@ -272,6 +272,11 @@ let _true = Ref(true), f, g, h
     @test_throws ErrorException h()
 end
 
+# Issue #33163
+A33163(x; y) = x + y
+@test try @code_typed A33163(1, y=2); true; catch; false; end
```

Comment:
The try/catch shouldn't be necessary. The test will fail if an error is thrown. Generally the pattern is just to check the return type or something in the test expr